### PR TITLE
Add settings ui

### PR DESCRIPTION
**Pull request by bugsalexander**
Web notifs not working, but it's because of an issue in master. I will fix in a separate PR.

### CODE REVIEW

File: packages/app/components/Nav/NavBar.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import Link from "next/link";
 import { API } from "@template/api-client";
 import useSWR from "swr";
 import { useProfile } from "../../hooks/useProfile";
+import Settings from "./Settings/Profile";
```
Comment (aislinblack) [inline packages/app/components/Nav/NavBar.tsx:11]: Is there a reason we are importing this? It seems like it isn't being used in this file

File: packages/app/components/Nav/RightNavBar.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import React, { Component } from "react";
 import styled from "styled-components";
-import { Avatar, Menu } from "antd";
+import { Avatar, Button, Menu } from "antd";
```
Comment (aislinblack) [inline packages/app/components/Nav/RightNavBar.tsx:3]: Are we using Button in this?

File: packages/app/components/Nav/Settings/Profile.tsx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+              <LableText> Phone </LableText>
+              <InputContainer>
+                {" "}
+                <Input
+                  value={phoneNumber}
+                  onChange={(newNum) => {
+                    setPhoneNumber(newNum.target.value);
+                  }}
+                  placeholder={"XXX-XXX-XXXX"}
+                />{" "}
```
Comment (aislinblack) [inline packages/app/components/Nav/Settings/Profile.tsx:141]: Do we need the {" "} in this file?

**Comment by willstenzel** [comment]
Does the settings page show on mobile? I looks as if the hamburger menu is covering the avatar icon at mobile widths

image

File: packages/app/components/Nav/NavBar.tsx (hunk 1 above)
Comment (bugsalexander) [reply packages/app/components/Nav/NavBar.tsx:11]: Good catch! I'm removing all unused imports

File: packages/app/components/Nav/RightNavBar.tsx (hunk 2 above)
Comment (bugsalexander) [reply packages/app/components/Nav/RightNavBar.tsx:3]: nope, you're right.

File: packages/app/components/Nav/Settings/Profile.tsx (hunk 3 above)
Comment (bugsalexander) [reply packages/app/components/Nav/Settings/Profile.tsx:141]: I removed them and the UI looked the same to me, so I don't think so. removing them.

File: packages/app/components/Nav/Settings/Profile.tsx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+              <SwitchContainer>
+                {" "}
+                <Switch
+                  checked={phoneNotifToggled}
+                  onChange={onPhoneNotifToggle}
+                />{" "}
+              </SwitchContainer>
+            </Row>
+            <Row>
+              <LableText> Phone </LableText>
```
[the remark this answers is not in the document]
Comment (bugsalexander) [reply packages/app/components/Nav/Settings/Profile.tsx:132]: yeet

File: packages/app/components/Nav/Settings/Profile.tsx [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+                  value={phoneNumber}
+                  onChange={(newNum) => {
+                    setPhoneNumber(newNum.target.value);
+                  }}
+                  placeholder={"XXX-XXX-XXXX"}
+                />{" "}
+              </InputContainer>
+            </Row>
+            <Row>
+              <Link href="/login" as="/login">
```
[the remark this answers is not in the document]
Comment (bugsalexander) [reply packages/app/components/Nav/Settings/Profile.tsx:145]: thanks for hte tip!

**Comment by bugsalexander** [comment]
@willstenzel I talked to @liustanley about that just now, and I think the icon is actually supposed to be a hamburger like that; however you did catch something else-  we're supposed to be displaying "profile" or "settings" inside the hamburger menu, and we are not.

**Comment by bugsalexander** [comment]
yep @dajinchu . the way olivia and i designed it was such that we display their phone number. of course, it doesn't need to be the case.

**Comment by bugsalexander** [comment]
ready to merge @liustanley

**Pull request merged**